src/services/config.rs

```diff
@@ -1,17 +1,20 @@
 use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Default, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub struct RpcConfig {
-    addr: String,
-    token: String,
+    pub enable: bool,
+    pub addr: String,
+    pub token: String,
+    pub duration: u64,
 }
 
-impl RpcConfig {
-    pub fn addr(&self) -> &str {
-        &self.addr
-    }
-
-    pub fn token(&self) -> &str {
-        &self.token
+impl Default for RpcConfig {
```

Comment:
The `Default` impl is useless because it will never establish a valid RPC connection.
Rust docs defines `Default` as "A trait for giving a type a useful default value", see: https://doc.rust-lang.org/std/default/trait.Default.html

Reply:
It's for generate a default `config.toml` file

Reply:
It's heavy to impl `Default` just for generating the default `config.toml` file.

Reply:
I will merge this since there is already a lot of `Default` impls just for docs gen. It might be better to add configuration templates in the future.